pyproject.toml

```diff
@@ -25,7 +25,7 @@ dependencies = [
     "pytest-cov",
 
     # required to compile older versions of vyper
-    "vvm",
+    "vvm>=0.3.1,<0.4.0",
```

Comment:
```suggestion
    "vvm>=0.3.1",
```

Reply:
I don't mind changing.. But why is that better?